### ISSUE TITLE
Unable to load Admob Ads under multiple configurations

### ISSUE THREAD

**Issue by pwrtoppl** [Scirra/Construct-3-bugs#398]
## Problem description

Unable to get Admob to load under multiple circumstances on multiple test devices

## Attach a .c3p

Any demo c3p works, just add a 'start on layout -> load ad (preload didn't fix it either, even with a wait command between)'

## Steps to reproduce

1. Add Admob through C3
2. Turn on/off test mode, add full screen ID info
3. Compile and install

## Observed result

No ad appears to run, under any circumstance. Admob dashboard shows no network calls being made

## Expected result

Either a test ad or actual ad per config

## Affected browsers

* Android: Yes
* iOS: Yes

## System details

Platform information
Browser: Chrome
Browser version: 58.0.3029.110
Browser engine: Blink
Browser architecture: 64-bit
Context: webapp
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 4
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36
C3 release: r32

Local storage
Storage quota (approx): 7.2 gb
Storage usage (approx): 358 mb (4.8%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

CSS element() is not supported.
Backdrop filters are not supported. Some UI effects are disabled.
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) HD Graphics 520 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

**Comment by AshleyScirra** [comment]
Our current plan is to entirely replace the plugin with a new one in C3. Iain is working on that.

**Comment by pwrtoppl** [comment]
I kind of figured when I was digging through the project files for iOS and found the cordova plugin missing and what looked like native c3 code referencing the Admob properties directly from the script itself.
Thanks for clearing that up!
Is there a chance that plugin will be ready once the licenses become available for puchase?

Thanks!

**Comment by AshleyScirra** [comment]
We can't give an ETA at this time I'm afraid, but hopefully within the first few weeks of licenses becoming available.

**Comment by pwrtoppl** [comment]
Hi @shortercode , I was hoping for maybe an update regarding the admob/gamecenter plugins for ios/android, I know Ashley mentioned it'd be a few weeks and I don't want to push the issue any, but some info would be greatly appreciated.

Thanks

**Comment by shortercode** [comment]
@pwrtoppl Unfortunately I've got a number of high priority difficult bugs to work on at the moment, so I can't really commit to a timescale. I'll can say I'm working on iOS stuff at the moment though.

**Comment by Parth-Kadakia** [comment]
Is this being fixed yet?

**Comment by odapplications** [comment]
This is a terriable bug. I have spent hours trying to get this feature to work. 

Can someone please provide a work around? or even a solution to admob? I just paid $100 and now I am seeing the investment go away....

**Comment by Parth-Kadakia** [comment]
Any update on this Bug?

**Comment by shortercode** [comment]
Okay so the new advert plugin has been released now, and the old one has been officially depreciated.

I did look into fixing the original plugin, for people already using it. But unfortunately the native plugin has a bug that prevents it from installing correctly. Given that the native plugin is third party we can't fix it directly, and the author has been unresponsive to previous bug reports we've filed with him on this plugin so it's unlikely to be fixed.

So the advice is, move to the new plugin. It shouldn't be too hard to switch over, I'm writing the manual entry at the moment.
